js/animate.js

```diff
@@ -41,19 +36,19 @@ const removeClassName = () => {
 };
 
 memberList.forEach((member) => {
-  member.style.setProperty(
-    '--s',
-    `${randomNumberWithinRange(0.1, 0.6) * responsiveConstant + 0.9}`,
-  );
+  member.style.setProperty('--s', `${randomNumberWithinRange(0.9, 1.8)}`);
 });
 
-window.addEventListener('scroll', () => {
-  let show = false;
-  if (!show && window.scrollY > height * 0.8 && window.scrollY < height * 1.8) {
-    show = true;
-    addClassName();
-  } else {
-    removeClassName();
-    show = false;
-  }
-});
+let handleIntersection = (entries) => {
+  entries.forEach((element) => {
+    if (element.isIntersecting) {
+      addClassName();
+    } else {
+      removeClassName();
+    }
```

Comment:
Since we are only using the `isIntersecting` property from the `element` object, let's destructure it and use it. Also let's use the shorthands for if-else

```javascript
entries.forEach(({ isIntersecting }) => {
 if (isIntersecting) addClassName();
 else removeClassName();
});
```

Now why I am telling you to do this?

- The code now looks concise and easier to read and understand.

Reply:
Yeah, I see...I knew this but forgot to implement it. Thanks for the reminder!👍

Reply:
Good catch @sumitd94 , Even I will remember and implement in my future PRs , thanks